resdk/resources/collection.py

```diff
@@ -185,3 +185,14 @@ def relations(self):
             self._relations = self.resolwe.relation.filter(collection=self.id)
 
         return self._relations
+
+    def duplicate(self):
+        """Duplicate (make copy of) ``collection`` object.
+
+        :return: Duplicated collection
+        """
+        if self.id is None:
```

Comment:
The concept of checking the existence of `id` was so common that we made a [assert_object_exists](https://github.com/genialis/resolwe-bio-py/blob/master/resdk/utils/decorators.py#L24) decorator for it. Please, use it here and in all other `duplicate()` methods.